lib/recursive_open_struct.rb

```diff
@@ -56,6 +57,8 @@ def recurse_over_array array
     end
   end
 
+  alias_method :array, :recurse_over_array
```

Comment:
This shouldn't be necessary. `recurse_over_array` does not refer to an attribute or instance variable, but is instead used internally to do the recursion step over Arrays. Unfortunately, `recurse_over_array` is a public method instead of being private --- this is something I'd change if there were other API-breaking changes in a new release.